microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -661,9 +661,14 @@ def _force_dpdk_default_source(self, variables: Dict[str, Any]) -> None:
     def after_case(self, log: Logger, **kwargs: Any) -> None:
         environment: Environment = kwargs.pop("environment")
         for node in environment.nodes.list():
+            interface = node.features[NetworkInterface]
+            if not interface.is_enabled_sriov():
+                log.debug("DPDK detected SRIOV was left disabled during cleanup.")
+                interface.switch_sriov(enable=True, wait=False, reset_connections=True)
             modprobe = node.tools[Modprobe]
+            # cleanup driver changes
             if modprobe.module_exists("uio_hv_generic"):
                 node.tools[Service].stop_service("vpp")
                 modprobe.remove(["uio_hv_generic"])
-                node.close()
```

Comment:
not remove this, since switch_sriov is not always executed?

Reply:
fixed, good catch!